compiler/rustc_codegen_cranelift/src/lib.rs

```diff
@@ -10,6 +10,7 @@
 #![warn(rust_2018_idioms)]
 #![warn(unused_lifetimes)]
 #![warn(unreachable_pub)]
+#![feature(box_patterns)]
```

Comment:
This feature was also added upstream, so this now conflicts: https://github.com/rust-lang/rust/blob/4b9f5cc4c10a161047475cb9bbe02c4fda57fb07/compiler/rustc_codegen_cranelift/src/lib.rs#L8